isilon_create_directories.sh

```diff
@@ -276,18 +276,17 @@ fi
 banner "Creates Hadoop directory structure on Isilon system HDFS."
 
 # Set permissions on HDFS root
-fixperm $HDFSROOT "hdfs$CLUSTER_NAME" "hadoop$CLUSTER_NAME" "755"
+fixperm "$HDFSROOT" "hdfs$CLUSTER_NAME" "hadoop$CLUSTER_NAME" "755"
 
-prefix=0
 # Cycle through directory entries comparing owner, group, perm
 # Sample output from "ls -dl"  command below
 # drwxrwxrwx    8 hdfs  hadoop  1024 Aug 26 03:01 /tmp
 
 for direntry in ${dirList[*]}; do
-   read -a specs <<<"$(echo $direntry | sed 's/#/ /g')"
+   read -ar specs <<<"${direntry//#/ }"
```

Comment:
This needs to be `-ra` (`specs` is the arg for `-a`). Also, this could use `IFS='#'`.